hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_unr_excl.el

```diff
@@ -319,14 +319,6 @@ Toggle 1to0 lc_otp_vendor_test_o.status [0] "logic lc_otp_vendor_test_o.status[3
 ANNOTATION: "VC_COV_UNR"
 Toggle 0to1 lc_otp_vendor_test_o.status [0] "logic lc_otp_vendor_test_o.status[31:0]"
 ANNOTATION: "VC_COV_UNR"
-Toggle 1to0 otp_alert_o.p "logic otp_alert_o.p"
-ANNOTATION: "VC_COV_UNR"
-Toggle 0to1 otp_alert_o.p "logic otp_alert_o.p"
-ANNOTATION: "VC_COV_UNR"
-Toggle 1to0 otp_alert_o.n "logic otp_alert_o.n"
-ANNOTATION: "VC_COV_UNR"
-Toggle 0to1 otp_alert_o.n "logic otp_alert_o.n"
-ANNOTATION: "VC_COV_UNR"
```

Comment:
@cindychip is it ok to remove these exclusions manually, or should we rerun UNR?

Reply:
you can move it. I will rerun UNR for V3  again :)